docs/resources/pages_project.md

```diff
@@ -174,7 +174,7 @@ Optional:
 - `fail_open` (Boolean) Fail open used for Pages Functions. Defaults to `false`.
 - `kv_namespaces` (Map of String) KV namespaces used for Pages Functions.
 - `r2_buckets` (Map of String) R2 Buckets used for Pages Functions.
-- `secrets` (Map of String) Encrypted environment variables for Pages Functions.
+- `secrets` (Map of String, Sensitive) Encrypted environment variables for Pages Functions.
```

Comment:
this part is also generated by the `make docs` cmd.  Perhaps it is necessary too. so I just leave this diff.

Reply:
this was due to your fork not having the latest commits. i updated the branch with the latest and it is now only showing those changes for this PR.